### DISCUSSION TITLE
Magma Core Orchestrator

### DISCUSSION THREAD

**Discussion by simransaxena08** [magma/magma#15231] [Q&A]
Hello all, I am planning to integrate Magma Core into our company's existing LTE setup. Before I move forward, I have some fundamental questions that I needed help with- 
1. Where is the access gateway exactly located? On the eNodeB device?
2. Where is the orchestrator located? Can it just be run on an AWS EC2 instance?
3. Do we need any other AWS instances running apart from the Orchestrator?
4. Do we need a VPN connection between the access gateway and orchestrator (i.e. the AWS EC2 instance and the eNodeB)?
5. Why can't we use the Amazon Machine Instance (AMI) anymore on AWS?
6. Is there a way to work with the Orchestrator without using Kubernetes? 

Thanks!

**Comment by javieraubert** [comment]
Hi, 
1- The AGW is a separate component, it can be located near the enb or on a remote location, but the enb S1 interface config needs to be pointed the the S1 interface of the AGW and have conectivity.
2- The orch are located on your cloud, can be your own servers or on a public cloud like AWS, Digital Ocean or whateaver cloud service you want.
3- If using AWS you just need the orch instances. Unless you need the feg, you will another instance for the feg.
4- The orch have some ports where the services are exposed, if you are not going to allow those ports to be available to the internet, then you will need to have a VPN.
5- Not sure, sorry.
6- Yes.

**Reply by simransaxena08** [reply]
Hello, 
Thank you so much for your prompt response! I was wondering if there is any documentation you can direct me to that could guide me on how to implement an Orchestrator without Kubernetes.

**Reply by javieraubert** [reply]
Hi, you could try this: https://magma.github.io/magma/docs/basics/quick_start_guide#terminal-tab-2-start-orchestrator
or this one wich is based on my own experience deploying the orchestrators without kubernetes: https://wiki.jaubertnet.com/books/orchestrators/page/deploying-the-orchestrators-v18

**Reply by simransaxena08** [reply]
Thanks a lot! Will try to follow these!

**Reply by Vinzclorth0** [reply]
Hey Javier,

Thanks for all the great information. I have set this up and am having trouble getting connections on 24224 from the AGWs -- Looking in the configs I see that fluentd is supposed to be listening on that port, but I don't see a container and I don't see anything listening on that port. Any ideas?

**Reply by javieraubert** [reply]
Hi @Vinzclorth0, you have to check the logs to see what is happening, you can do docker ps -a to see if the fluentd container exited, if yes you can do docker logs "fluentd_containers_name" to see the containers logs.
